ROMFS/px4fmu_common/mixers/tilt_quad.aux.mix

```diff
@@ -0,0 +1,38 @@
+Tilt-Quadrotor mixer for PX4FMU (2/2) V2.0
+===========================
+
+This file defines the aux outputs mixer for a Tilt-quadrotor in the + configuration.
+
+
+# @output AUX1 servo motor 1
```

Comment:
Can you give these meaningful names? Aileron 1, 2, Elevator, etc.
https://docs.px4.io/en/airframes/airframe_reference.html#standard-vtol

Reply:
I will change them for something more specific